src/routes/(site)/teams/+page.svelte

```diff
@@ -32,6 +32,7 @@
         gateways for students to explore new fields, develop new interests, and learn new skills
         that will benefit them in the industry.
       </p>
+      <p><span style="color:red">*</span>Feel free to reach out to board members through their discord, stated below their profile.</p>
```

Comment:
```suggestion
      <p>Feel free to reach out to board members through their Discord username, stated below their profile.</p>
```

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1099/commits/044a292ec9dac6d99f9e07edbed81caf12424f52>.